package/yast2-registration.spec

```diff
@@ -43,7 +43,8 @@ Requires:       SUSEConnect
 
 Requires:       yast2-slp >= 3.1.2
 Requires:       yast2-add-on >= 3.1.8
-Requires:       yast2-packager >= 3.1.26
+# packager/product_patterns.rb
+Requires:       yast2-packager >= 3.1.95
```

Comment:
3.1.95 introduced the `packager/product_patterns.rb` file, 3.1.97 just changed the place where the new code was used in the packager itself. The API haven't changed since 3.1.95, so it's the correct version.

(Anyway, for the end users it's irrelevant, both 3.1.95 and 3.1.97 were not released to public, they will see 3.1.98+ :wink:)